lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -90,17 +90,22 @@ defmodule Mix.Tasks.Xref do
       with at least one transitive dependency. See "Dependencies types"
       section below.
 
+    * `--group` - provide pipe-separated paths to consider as merged into
+      a single group. The group is named after the first path, with a `+` suffix.
+      Can be repeated to create multiple groups.
```

Comment:
Clarifying a little bit...

```suggestion
    * `--group` - provide pipe-separated paths to consider as a group. Dependencies
      from and into multiple files of the group are considered a single dependency.
      Dependencies between the group elements are ignored. This is useful when you
      are computing compile and compile-connected dependencies and you want a
      series of files to be treated as one. The group is printed using the first path,
      with a `+` suffix. Can be repeated to create multiple groups.
```